stellar-common/src/payments.rs

```diff
@@ -26,17 +26,37 @@ fn map_payments(block: Block) -> Result<Payments, substreams::errors::Error> {
             Err(_) => return,
         };
 
+        // You may want to consider the fees, which appear at the transaction
+        // level, as a payment, since there will be a debit from the payer's
+        // account, though there is no destination (fees are deposited into an
+        // inaccessible fee pool).
```

Comment:
I think we could deduct the fees from the transaction / operation itself and use that last amount